cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -2348,37 +2346,41 @@ fn x64_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut OperandCol
             collector.reg_early_def(*tmp);
         }
 
-        Inst::CallKnown { dest, ref info, .. } => {
+        Inst::CallKnown { dest, info, .. } => {
             // Probestack is special and is only inserted after
             // regalloc, so we do not need to represent its ABI to the
             // register allocator. Assert that we don't alter that
             // arrangement.
-            debug_assert_ne!(*dest, ExternalName::LibCall(LibCall::Probestack));
-            for u in &info.uses {
-                collector.reg_fixed_use(u.vreg, u.preg);
-            }
-            for d in &info.defs {
-                collector.reg_fixed_def(d.vreg, d.preg);
+            if let Some(info) = info {
```

Comment:
Some thoughts that I had while changing this:

* In case there are no objections or downsides to making `BoxCallInfo` optional, would it be preferred to expect that the info is present in this path?
* Another alternative that I considered in case that making `BoxCallInfo` optional is not desired, would be to create another instruction without `BoxCallInfo`, which would be only consumed by Winch, at the cost of some code duplication.

Reply:
Thanks for the suggestion! Added a comment.